### ISSUE TITLE
Replace is not Update

### ISSUE THREAD

**Issue by Fruchtgummi** [diegogub/aranGO#46]
I miss Update-Function! Replace need all data, update can also single value and does not consider the other values. With several objects in a collection is unproductive. With all due respect! 

`// update document
  d1.Age = 23
  err =s.DB("test").Col("docs1").**Replace**(d1.Key,d1)
  if err != nil {
    panic(err)
  }`

**Comment by diegogub** [comment]
@Fruchtgummi you are right, I also have Patch() function. I will include it in next version!
thanks
